openmc/plots.py

```diff
@@ -650,6 +651,42 @@ def to_xml_element(self):
 
         return element
 
+    def to_image(self, openmc_exec='openmc', cwd='.', convert_exec='convert'):
+        """Render plot as an image
```

Comment:
I find `to_image` a little vague.  How about `render_ipython_image`?

Reply:
I'm trying to stick to the convention of `to_*` meaning "convert this to another type" which we use in many other places in the API (note the difference between that and `export_to_*` which implies that a file is written). I can rename it to `to_ipython_image` though; how about that?

Reply:
Sure, `to_ipython_image` sounds good.  Although I'd also like you to mention in the docstring that this is calling OpenMC (to make it clear this is a potentially expensive function that requires all settings, geometry, and materials XMLs to already be exported).